pkg/config/config.go

```diff
@@ -292,55 +303,99 @@ func httpClientForIssuer(iss OIDCIssuer) (*http.Client, error) {
 }
 
 func (fc *FulcioConfig) prepare() error {
-	if _, ok := fc.GetIssuer("https://kubernetes.default.svc"); ok {
-		// Add the Kubernetes cluster's CA to the system CA pool, and to
-		// the default transport.
+	fc.verifiers = make(map[string][]*verifierWithConfig, len(fc.OIDCIssuers))
+	for _, iss := range fc.OIDCIssuers {
+		if err := fc.insertVerifier(iss); err != nil {
+			log.Logger.Errorf("error creating provider for issuer URL %q: %v", iss.IssuerURL, err)
+			continue
+		}
+	}
+
+	_, verifierPresent := fc.verifiers[k8sIssuerURL]
+	k8sIssuer, issuerConfigured := fc.GetIssuer(k8sIssuerURL)
+	if issuerConfigured && !verifierPresent {
+		// configure static validator for k8s that cover metaIssuers
+		if err := fc.insertVerifier(k8sIssuer); err != nil {
+			log.Logger.Errorf("error creating provider for issuer URL %q: %v", k8sIssuer.IssuerURL, err)
+		}
+	}
+
+	cache, err := lru.New2Q[string, []*verifierWithConfig](100 /* size */)
+	if err != nil {
+		return fmt.Errorf("lru: %w", err)
+	}
+	fc.lru = cache
+	return nil
+}
+
+var (
```

Comment:
These could all be consts.

Reply:
Yes, they could, but I make them `vars` because it allows me to change the value for testing purposes. https://github.com/bouskaJ/fulcio/blob/fix_2110/pkg/config/config_test.go#L905